pkg/kando/location_push.go

```diff
@@ -17,16 +17,10 @@ package kando
 import (
 	"context"
 	"io"
-	"os"
 
-	"github.com/pkg/errors"
-	"github.com/spf13/cobra"
-
-	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
-	"github.com/kanisterio/kanister/pkg/kopia/snapshot"
 	"github.com/kanisterio/kanister/pkg/location"
-	"github.com/kanisterio/kanister/pkg/output"
 	"github.com/kanisterio/kanister/pkg/param"
+	"github.com/spf13/cobra"
```

Comment:
lets move this to a separate group of imports

Reply:
Sure !